andela/wg-shleemypants
# Feature: show info exercise 157110615

**What does this PR do?**
It creates a private API endpoint for viewing exercise details. For example, categories, muscles and images
**Description of tasks completed**
- Creates a class serializer for all the exercise details
- Creates and registers the url for the endpoint
- Puts the endpoint in private endpoints templates
- creates tests for the endpoint

**How can this be manually tested**

- `git clone https://github.com/andela/wg-shleemypants.git`
- `git checkout ft-show-info-exercise-157110615`
- `pip install -r requirements.txt`
- Follow other setup configurations in the main `readme`
- `python manage.py runserver`
- open browser and navigate to `http://127.0.0.1:8000/api/v2/exercisedetails/`
- All the details of the exercise should show up

**Screenshots**

<img width="756" alt="screen shot 2018-05-04 at 12 47 15 pm" src="https://user-images.githubusercontent.com/13331207/39622615-99b82e14-4f9b-11e8-9ff7-35c6df8a4b51.png">

**Related pivotal tracker story id**

[157110615](https://www.pivotaltracker.com/story/show/157110615)
